lib/oli_web/templates/layout/_footer.html.eex

```diff
@@ -14,4 +13,7 @@
       </div>
     </div>
   </div>
-</footer>
+  <script>
```

Comment:
I wanted to eliminate this call (and the one for delivery mode above) back to the server since it will happen on every single page render, regardless of whether the user ever opens the "Cookie Preferences" page.

Reply:
Server call now blocked if opt-in cookie present in the browser